project/settings.py

```diff
@@ -51,7 +51,7 @@ def get(variable):
 
     'apps.file',
     'apps.socialmedia',
-    'apps.support',
+    'apps.logs',
```

Comment:
This is a great PR. I love how it's not messing with the user app by default. But, I can't accept an app that is plural when none of the others are 😆 maybe `apps.logging`?

Reply:
Alternate proposal: rename _everything_ to be plural (`apps.files`, `apps.socialmediums`, `apps.users`...)